lib/universal/mapper.js

```diff
@@ -70,12 +70,16 @@ exports.track = function(track, settings) {
   var result = createPageDataForm(track, createCommonGAForm(track, settings));
   var library = track.proxy('context.library');
   var screenName = track.proxy('properties.screenName');
+  var url = track.proxy('context.page.url') || track.proxy('properties.url');
+  var library = track.proxy('context.library');
+  var screenName = track.proxy('properties.screenName');
```

Comment:
@f2prateek , I think properties.screenName should be added to our mobile spec. For mobile apps, GA requires "Screen" names to be passed with every track event, but we're currently not passing any because we don't collect screen name as a common context property. On web, we currently pass "Page" name to GA via the "dp" field in our GA track requests and those page names populate in the GA dashboard here: https://cloudup.com/c3Ovs9Zldqh. They are grabbed from context.page.path automatically. The problem is we do not provide "Screen" names for mobile events the way we provide "Page" names for web events. "Screen" names should be passed via the "cs" field in our GA track requests, but we are currently leaving it undefined. The result for the end user is that the "Screen" column will remain unpopulated within their GA Mobile dashboards. When they look at this(https://cloudup.com/c3Ovs9Zldqh) screen on their GA mobile dashboard, they will see a "Screen" column instead of a "Page" column, and their screen names will all be blank. Since seeing screen names attached to events is a pretty basic need for mobile GA users, I think we should include properties.screenName in our mobile spec - otherwise we have to allow users to include it as a GA specific options parameter.